mne/forward/_make_forward.py

```diff
@@ -53,58 +51,61 @@ def _read_coil_defs(fname=None, elekta_defs=False, verbose=None):
         cosmag contains the direction of the coils and rmag contains the
         position vector.
     """
-    if fname is None:
-        if not elekta_defs:
-            fname = op.join(op.split(__file__)[0], '..', 'data',
-                            'coil_def.dat')
-        else:
-            fname = op.join(op.split(__file__)[0], '..', 'data',
-                            'coil_def_Elekta.dat')
+    coil_dir = op.join(op.split(__file__)[0], '..', 'data')
+    coils = list()
+    if elekta_defs:
+        coils += _read_coil_def_file(op.join(coil_dir, 'coil_def_Elekta.dat'))
+    coils += _read_coil_def_file(op.join(coil_dir, 'coil_def.dat'))
```

Comment:
Can you add add a note to the `elekta_defs` docstring entry in case someone else ends up looking at or reusing this function for their own purposes? Repeated definitions probably won't be expected...

Something along the lines of:
`If true, prepend Elekta's coil definitions for numerical integration (from Abramowitz and Stegun section 25.4.62). Note that this will likely cause duplicate coil definitions.`